Translations/pt_BR.po

```diff
@@ -875,359 +871,306 @@ msgstr ""
 msgid "Read-Only Directory Error"
 msgstr "Erro de diretório, somente leitura"
 
-#: Source/automap.cpp:484
-msgid "game: "
-msgstr "jogo: "
+#: Source/automap.cpp:485
+msgid "Game: "
+msgstr "Jogo: "
 
-#: Source/automap.cpp:490
-msgid "password: "
-msgstr "senha: "
+#: Source/automap.cpp:491
+msgid "Password: "
+msgstr "Senha: "
 
-#: Source/automap.cpp:492
-#, fuzzy
-#| msgid "Quit Game"
+#: Source/automap.cpp:493
 msgid "Public Game"
-msgstr "Sair do jogo"
+msgstr "Jogo Público"
 
-#: Source/automap.cpp:504
+#: Source/automap.cpp:505
 msgid "Level: Nest {:d}"
 msgstr "Nível: Enxame {:d}"
 
-#: Source/automap.cpp:506
+#: Source/automap.cpp:507
 msgid "Level: Crypt {:d}"
 msgstr "Nível: Cripta {:d}"
 
-#: Source/automap.cpp:508 Source/items.cpp:2090
+#: Source/automap.cpp:509 Source/items.cpp:2034
 msgid "Level: {:d}"
 msgstr "Nível: {:d}"
 
-#: Source/control.cpp:212
+#. TRANSLATORS: {:s} means: Game Difficulty.
+#: Source/automap.cpp:527
+msgid "Difficulty: {:s}"
+msgstr "Dificuldade: {:s}"
+
+#: Source/control.cpp:152
 msgid "Tab"
 msgstr "Tab"
 
-#: Source/control.cpp:212
+#: Source/control.cpp:152
 msgid "Esc"
 msgstr "Esc"
 
-#: Source/control.cpp:212
+#: Source/control.cpp:152
 msgid "Enter"
 msgstr "Enter"
 
-#: Source/control.cpp:215
+#: Source/control.cpp:155
 msgid "Character Information"
 msgstr "Informações do personagem"
 
-#: Source/control.cpp:216
+#: Source/control.cpp:156
 msgid "Quests log"
 msgstr "Registro de missões"
 
-#: Source/control.cpp:217
+#: Source/control.cpp:157
 msgid "Automap"
 msgstr "Automapa"
 
-#: Source/control.cpp:218
+#: Source/control.cpp:158
 msgid "Main Menu"
 msgstr "Menu Principal"
 
-#: Source/control.cpp:219
+#: Source/control.cpp:159 Source/diablo.cpp:1458
 msgid "Inventory"
 msgstr "Inventário"
 
-#: Source/control.cpp:220
+#: Source/control.cpp:160
 msgid "Spell book"
 msgstr "Livro de feitiços"
 
-#: Source/control.cpp:221
+#: Source/control.cpp:161
 msgid "Send Message"
 msgstr "Enviar Mensagem"
 
-#: Source/control.cpp:718 Source/control.cpp:1587
-msgid "Skill"
-msgstr "Habilidade"
+#: Source/control.cpp:662
+msgid "Player friendly"
+msgstr "Amigável a jogadores"
+
+#: Source/control.cpp:664
+msgid "Player attack"
+msgstr "Hostil a jogadores"
+
+#: Source/control.cpp:667
+msgid "Hotkey: {:s}"
+msgstr "Tecla de atalho: {:s}"
+
+#: Source/control.cpp:675
+msgid "Select current spell button"
+msgstr "Selecionar botão do feitiço atual"
```

Comment:
This one feels incorrect to me. I believe the direct translation should be:
```suggestion
msgstr "Botão para selecionar feitiço atual"
```
However, that obviously looks terrible. I think we should seriously reconsider the original text here and remove the "button" part altogether, then translate as
```suggestion
msgstr "Selecionar feitiço atual"
```